docs/process-services-cloud/components/people-cloud.component.md

```diff
@@ -29,6 +29,7 @@ Allows one or more users to be selected (with auto-suggestion) based on the inpu
 | preSelectUsers | [`IdentityUserModel`](../../../lib/core/models/identity-user.model.ts)`[]` |  | Array of users to be pre-selected. All users in the array are pre-selected in multi selection mode, but only the first user is pre-selected in single selection mode. Mandatory properties are: id, email, username |
 | roles | `string[]` |  | Role names of the users to be listed. |
 | searchUserCtrl | `FormControl` | new FormControl() | FormControl to search the user |
+| removeLastChip | `Boolean` | true | This flag enables not to delete last chip in multiple mode. In case the flag is false, last chip from the list can not be deleted  Otherwise, last chip can be deletable. |
```

Comment:
I don't like this strategy. As I said we should be able to pass a readonly property(true/false) for the preselected users

Reply:
Done